content/docs/developing-operators/plans.md

```diff
@@ -122,6 +122,8 @@ spec:
                 - cleanup
 ```
 
+As you see, the `cleanup` plan is basically a "negative" (`kind: Delete`) to the `deploy` (`kind: Apply`) plan - it simply deletes the previously applied resources. Note, that a `Delete` task does not fail if the resources do not exist.
```

Comment:
For a negative of the `deploy` plan one wouldn't need a `cleanup` plan: Resources deployed on the cluster by an `Instance` will be deleted when the `Instance` is being deleted. A `cleanup` plan is meant for actions that cannot easily be negated because they are specific to the operator. E.g., if you need to run a job that informs an external service about the termination of an instance.